### ISSUE TITLE
Performance issue in the definition of train, lessonOne/imgClassifierWeb/execute.py(P1)

### ISSUE THREAD

**Issue by DLPerf** [zhaoyingjun/TensorFlow-Coding#13]
Hello, I found a performance issue in the definition of `train`, lessonOne/imgClassifierWeb/execute.py, [`tf.argmax(softmax_propabilities, axis=1)`](https://github.com/zhaoyingjun/TensorFlow-Coding/blob/1ff292e5d659aa98e7bf6d9cc3986ef07ac2ca81/lessonOne/imgClassifierWeb/execute.py#L151) will be created repeatedly during program execution, resulting in reduced efficiency. I think it should be created before the loop.

The same issue exists in:

- [`tf.argmax(softmax_propabilities, axis=1)` in line 146](https://github.com/zhaoyingjun/TensorFlow-Coding/blob/1ff292e5d659aa98e7bf6d9cc3986ef07ac2ca81/lessonSix/vgg16/execute.py#L146)

Looking forward to your reply. Btw, I am very glad to create a PR to fix it if you are too busy.
